settings.gradle.kts

```diff
@@ -15,5 +15,6 @@ include("exposed-crypt")
 pluginManagement {
     plugins {
         id("org.jetbrains.kotlin.jvm") version "1.8.22"
+        id("org.jetbrains.kotlin.plugin.serialization") version "1.8.22"
```

Comment:
I believe that the serialization plugin must always match the kotlin plugin version. If true, does something need to be done with dependabot to ensure they don't diverge?

Reply:
That's true